lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -47,13 +48,13 @@ class CanvasKitRenderer implements Renderer {
   DomElement? _sceneHost;
   DomElement? get sceneHost => _sceneHost;
 
-  @visibleForTesting
   late Rasterizer rasterizer = Rasterizer();
 
   set resourceCacheMaxBytes(int bytes) => rasterizer.setSkiaResourceCacheMaxBytes(bytes);
 
   @override
   Future<void> initialize() async {
+    _instance = this;
```

Comment:
Maybe I'm being too cautious, but maybe this assignment should be the last thing in this function? I'm thinking it could prevent subtle bugs where something tries to access an uninitialized instance of `CanvasKitRenderer`.